core/io/resource.cpp

```diff
@@ -522,6 +515,30 @@ String Resource::get_id_for_path(const String &p_path) const {
 }
 #endif
 
+#ifdef TOOLS_ENABLED
+LocalVector<ConfigurationInfo> Resource::get_configuration_info() const {
+	LocalVector<ConfigurationInfo> ret;
+
+	Array info;
+	if (GDVIRTUAL_CALL(_get_configuration_info, info)) {
+		for (int i = 0; i < info.size(); i++) {
```

Comment:
You can resize the `ret` beforehand to avoid appending.
Also you can iterate Array with range iterator.